spdx/document.py

```diff
@@ -40,7 +41,7 @@ def from_identifier(cls, identifier):
         the full_name is retrived from it. Otherwise
```

Comment:
Do not hesitate to fix typos as you go: this should be "retrieved" , not "retrived"